notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb

```diff
@@ -1249,9 +1183,9 @@
       "source": [
         "### Get the latest state of the production endpoint\n",
         "\n",
-        "Now that the pipeline has finished, the contender (version 3) model has replaced the previous champion model on the production endpoint.\n",
+        "Once your pipeline execution is finished, you can notice that the contender (version 3) model has replaced the previous champion model on the production endpoint.\n",
```

Comment:
suggested edit:

-  "Once your pipeline execution is finished, notice that the contender (version 3) model has replaced the previous champion model on the production endpoint."